gateway/src/config/template.rs

```diff
@@ -34,6 +34,9 @@ private_sphinx_key_file = '{{ gateway.private_sphinx_key_file }}'
 # Path to file containing public sphinx key.
 public_sphinx_key_file = '{{ gateway.public_sphinx_key_file }}'
 
+# Addess to an Ethereum full node.
+eth_endpoint = '{{ gateway.eth_endpoint }}'
```

Comment:
Similar issue as with both clients, are we sure it won't blow up due to feature locking action?

Reply:
Tested it on the gateway as well, it works just fine, these fields are just left empty.